locallib.php

```diff
@@ -346,3 +346,83 @@ function ojt_can_evaluate($userid, $context) {
 
     return true;
 }
+
+/**
+ * Reorder topics and topic items in edit mode.
+ *
+ * @param $items
+ * @param $oldpos
+ * @param $newpos
+ * @return false
+ */
+function array_move(&$items, $oldpos, $newpos) {
```

Comment:
grrr...
https://docs.moodle.org/dev/Frankenstyle#Function_names

:-)

Reply:
thanks, I've renamed it